src/bionics.cpp

```diff
@@ -910,7 +910,7 @@ void player::process_bionic(int b)
 
     // Bionic effects on every turn they are active go here.
     if (bio.id == "bio_night") {
-        if (calendar::turn % 5) {
+        if (calendar::is_time_for(5)) {
```

Comment:
I hadn't caught that strings that are only used once were being extracted, you shouldn't extract any strings that are unique, it just adds extra lines for no benefit.  Magic numbers that are used for game logic are a bit different and I'm ok with them being extracted even if only used once.